notebooks/speechbrain-emotion-recognition/speechbrain-emotion-recognition.ipynb

```diff
@@ -0,0 +1,454 @@
+{
```

Comment:
<p>Maybe it is good to add some comparison with original model by benchmark_app and some demo with gradio with possibility of trying a custom file ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2064/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Will plan it for next release (with gradio).